docs/API/metrics_api.md

```diff
@@ -0,0 +1,155 @@
+## <a name="top"></a>Metrics API
+
+### Introduction
+
+The Perseo Metrics API is a REST-based API that can be used to get relevant operational metrics.
+
+[Top](#top)
+
+### Operations
+
+#### Get metrics
+
+```
+GET /admin/metrics
+```
+
+The response payload is a multi-level JSON tree storing the information in an structured way. This structure is based on
+service and subservice (sometimesrefered to as "service path"). At any point of the tree, the value of a key could be
+`{}` to mean that there isn't actual information associated to that key.
+
+At the first level there are two keys: **services** and **sum**. In sequence, **services** value is an object whose keys
+are service names and whose values are objects with information about the corresponding service. The **sum** value is an
+object with information for the aggregated information for all services.
+
+```
+{
+  "services": {
+    "service1": <service 1 info>,
+    "service2": <service 2 info>,
+    ...
+    "serviceN": <service N info>
+  }
+  "sum": <aggregated info for all services>
+}
+```
+
+Regarding service information objects, they use two keys: **subservs** and **sum**. In sequence, **subservs** value is
+an object whose keys are subservice names and whose values are objects with information about the corresponding
+subservice. The **sum** value is an object with information for the aggregated information for all subservices in the
+given services.
+
+```
```

Comment:
Add code highlight - similarly for  codeblocks found below

Reply:
Ok. Fixed in a0154d378be6652e9e0ec740668aece1bc4b4d63